packages/bootleg-app/src/CurrentUserCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { drizzleConnect } from 'drizzle-react';
+import { AccountData } from 'drizzle-react-components';
+import { Card } from 'rimble-ui';
+
+const CurrentUserCard = ({ accounts, drizzleStatus }) => (
+  <Card fontSize={4}>
+    <span>You are logged in with the following account:</span>
+    <AccountData
+      accountIndex="0"
+      units="ether"
+      precision="3"
+      render={({ address, balance, units }) => (
+        <div>
+          <div>
```

Comment:
Probably want a `Text` here maybe a `Text.p`

Reply:
Current user card I might remove... it's for demo/debugging. But if I don't, I'll fix it up nice.